src/index.js

```diff
@@ -115,6 +112,29 @@ const checkParams = (params) => {
   }
 };
 
+/**
+ * Normalizes a `refNode` param to an element - or `null`.
+ *
+ * @private
+ * @param  {Element|string} refNode
+ *         The value of a `refNode` param.
+ *
+ * @return {Element|null}
+ *         A DOM element or `null` if the `refNode` was given as a string and
+ *         did not match an element.
+ */
+const resolveRefNode = (refNode) => {
+  if (isElInDom(refNode)) {
+    return refNode;
+  }
+
+  if (typeof refNode === 'string') {
+    return document.querySelector(refNode);
+  }
+
+  return null;
+};
```

Comment:
`resolveRefNode` was moved out of the `create-embed` module. And it doesn't really need to throw if we only use it in one place (in the `loadPlayer` call stack) because it is then checked by `checkParams` and a more useful error is surfaced.